src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/common/DashboardStepDef.scala

```diff
@@ -19,11 +19,12 @@ package uk.gov.hmrc.test.ui.cucumber.stepdefs.common
 import uk.gov.hmrc.test.ui.cucumber.stepdefs.BaseStepDef
 import uk.gov.hmrc.test.ui.pages.common.DashboardPage
 
-class DashboardPageStepDef extends BaseStepDef {
+class DashboardStepDef extends BaseStepDef {
 
   And("""^I should land on Dashboard page""")(() => DashboardPage.checkPage())
 
   And("""^I validate declaration details on (.*) tab and check Status is (.*)""") { (tab: String, status: String) =>
+    DashboardPage.refreshPage()
```

Comment:
why do we need refresh here when you already have status refresh inside this validate method

Reply:
No, while validating the details, first time it is not showing the actual result in the dashboard page, so when we refresh the page it is getting the updated status

Reply:
removed now